builtin/logical/pki/path_acme_eab.go

```diff
@@ -115,7 +125,6 @@ a warning that it did not exist.`,
 type eabType struct {
 	KeyID         string    `json:"-"`
 	KeyType       string    `json:"key-type"`
-	KeyBits       int       `json:"key-bits"`
 	PrivateBytes  []byte    `json:"private-bytes"`
 	AcmeDirectory string    `json:"acme-directory"`
```

Comment:
While we have time to change this, should we add `/directory` here so it _is_ the actual directory we'd use with clients?

I know that makes validation a little harder in some places... I dunno?

Reply:
Done added in https://github.com/hashicorp/vault/pull/20945/commits/ab350e29f15442810d89c60e60b787eb53528e8f

Reply:
Thank you!!